core/chain_manager.go

```diff
@@ -48,6 +48,9 @@ var (
 )
 
 const (
+	headerCacheLimit    = 512
+	bodyCacheLimit      = 256
+	tdCacheLimit        = 1024
```

Comment:
Sure, but we have to start somewhere. And if they are only good for benchmarking imports then I'm sure we can setup a benchmark with 2-3 nodes where some are importing and others are publishing new blocks / txs, etc. Then we'd have a starting point for simulated scenarios where we can benchmark the effect of each cache and it's size.